test/uuid_test.cpp

```diff
@@ -1,42 +1,38 @@
-/*
- * uuid.cpp
- *      Author: carlos
- */
-
-#define CATCH_CONFIG_MAIN  // This tells Catch to provide a main() - only do this in one cpp file
+#define DOCTEST_CONFIG_IMPLEMENT_WITH_MAIN
 
+#include "doctest.h"
 #include "Uuid.h"
-#include "catch.hpp"
 #include <string>
 
 constexpr int MAX_ITERS = 100;
 
-TEST_CASE( "Uuid", "[uuid]" ) {
+TEST_CASE( "Uuid" ) {
   for (int i=0; i<MAX_ITERS; i++) {
     ids::Uuid uuid;
     std::string uuid_str {uuid.to_str()};
 
-    INFO(uuid_str);
-    REQUIRE(uuid.most > 0);
-    REQUIRE(uuid.least > 0);
-    REQUIRE(uuid_str.size() == 36);
+    MESSAGE(uuid_str);
+    CHECK(uuid.most > 0);
```

Comment:
CHECK in Doctest is equivalent to REQUIRE in Catch

Reply:
Hmmm... CHECK should have the same semantics in both frameworks - and REQUIRE too (REQUIRE terminates the test case with a throw statement when it fails)

Reply:
So I can examples code using either CHECK or REQUIRE in both (Catch or doctest), I will change it. Thanks!